certification/runtime/config.go

```diff
@@ -51,24 +52,36 @@ func NewConfigFrom(vcfg viper.Viper) (*Config, error) {
 	cfg.DockerConfig = vcfg.GetString("dockerConfig")
 	cfg.Artifacts = vcfg.GetString("artifacts")
 	cfg.WriteJUnit = vcfg.GetBool("junit")
-	cfg.storeContainerPolicyConfiguration(vcfg)
 	cfg.storeOperatorPolicyConfiguration(vcfg)
+
+	err := cfg.storeContainerPolicyConfiguration(vcfg)
+	if err != nil {
+		return nil, err
+	}
+
 	return &cfg, nil
 }
 
 // storeContainerPolicyConfiguration reads container-policy-specific config
 // items in viper, normalizes them, and stores them in Config.
-func (c *Config) storeContainerPolicyConfiguration(vcfg viper.Viper) {
+func (c *Config) storeContainerPolicyConfiguration(vcfg viper.Viper) error {
 	c.PyxisAPIToken = vcfg.GetString("pyxis_api_token")
 	c.Submit = vcfg.GetBool("submit")
 	c.PyxisHost = pyxisHostLookup(vcfg.GetString("pyxis_env"), vcfg.GetString("pyxis_host"))
 
 	// Strip the ospid- prefix from the project ID if provided.
 	certificationProjectID := vcfg.GetString("certification_project_id")
 	if strings.HasPrefix(certificationProjectID, "ospid-") {
+		// counting the `-` values, if the user provides more than one, we know they inputted a legacy project id
+		// which can not be used to query pyxis, exiting early to show the help menu
+		if strings.Count(certificationProjectID, "-") != 1 {
+			return errors.Errorf("certification prodject id: %s is improperly formatted see help for instructions on obtaining proper value", certificationProjectID)
+		}
 		certificationProjectID = strings.Split(certificationProjectID, "-")[1]
```

Comment:
Have you thought about still doing the strings.Split first, then just checking for the length of the returned slice? If it's > 2, then you would know you have more than one '-'. This would remove the extra scan of the string.